regen-network/regen-web
# Set up `v4` branch for Regen Ledger v4.0 upgrade

## Description

Closes: regen-network/regen-registry#983
Needs: https://github.com/regen-network/registry-server/pull/208/



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- ~~[ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)~~ not applicable here

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

Nothing much to test yet.
Once the server PR is merged, just make sure that the ledger related features don't work anymore on the registry deploy preview and that we're using /v4-ledger and /v4-ledger-rest proxy endpoints...

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
